src/pages/about-details/styled.js

```diff
@@ -0,0 +1,128 @@
+import styled from 'styled-components'
+
+export const StyledAbout = styled.div`
+  max-width:auto;
+  max-height:auto;
+  z-index: 10;
+  background-color: ${props => props.theme === 'about' ? '#FFFFFF' : '#fff'};
```

Comment:
Este ternário sempre resultará na mesma cor (`#FFFFFF` e `#FFF` são equivalentes), ele é necessário?

```suggestion
  background-color: #FFFFFF
```